libs/design/scss/layout/_breakpoint.scss

```diff
@@ -1,15 +1,21 @@
 @use 'sass:map';
 @use 'variables' as v;
 
-//
-// Take a the list of breakpoints and retrieves the defined point.
-// If an invalid breakpoint is defined, it will display a warning.
-//
-// @usage
-// ```
-// @include layout.breakpoint(mobile) {}
-// ```
-//
+/// Applies styles at a specific breakpoint defined by `$breakpoints`.
```

Comment:
Can you add a link to these?

Reply:
How should I link to another sass doc? There's currently no working direct path to `$breakpoints`.